### ISSUE TITLE
Screen readers refer to the search icon in the header as “button”

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7174]
### Where To Find Bug
In the navbar of all pages

### Expected Behaviour
The screen reader should announce the accessible name/functionality of the icon.

### Actual Behaviour
The screen reader refers to the icon as button.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Press tab until tab-focus is on the search icon in the header
- Notice that the screen reader refers to the icon as button

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
